src/modules/window/wrap_Window.cpp

```diff
@@ -94,6 +94,12 @@ static int readWindowSettings(lua_State *L, int idx, WindowSettings &settings)
 		settings.y = (int) luaL_optinteger(L, -1, 0);
 	}
 	lua_pop(L, 2);
+	lua_getfield(L, idx, settingName(Window::SETTING_TITLE));
+	if (!lua_isnoneornil(L, -1))
+	{
+		settings.title = luaL_checkstring(L, -1);
+	}
```

Comment:
As per our [code style](https://love2d.org/wiki/Code_Style), braces should be omitted whenever possible.
```suggestion
		settings.title = luaL_checkstring(L, -1);
```